plugins/auth-node/src/oauth/types.ts

```diff
@@ -44,6 +44,7 @@ export interface OAuthAuthenticatorAuthenticateInput {
 /** @public */
 export interface OAuthAuthenticatorRefreshInput {
   scope: string;
+  audience?: string;
```

Comment:
I don't think we should change this type just for pinniped; our refresh method will still have access to the `req` parameter anyway
```suggestion
```